diff -up package/lib/configure.js.externalgyp package/lib/configure.js
--- package/lib/configure.js.externalgyp	2015-08-19 08:37:29.003680036 +0200
+++ package/lib/configure.js	2015-08-19 08:38:01.083654927 +0200
@@ -313,7 +313,7 @@ function configure (gyp, argv, callback)
     argv.push('-Dnode_root_dir=' + nodeDir)
     argv.push('-Dmodule_root_dir=' + process.cwd())
     argv.push('--depth=.')
-    argv.push('--no-parallel')
+    //argv.push('--no-parallel')
 
     // tell gyp to write the Makefile/Solution files into output_dir
     argv.push('--generator-output', output_dir)
@@ -328,7 +328,7 @@ function configure (gyp, argv, callback)
     argv.unshift(gyp_script)
 
     // make sure python uses files that came with this particular node package
-    process.env.PYTHONPATH = path.resolve(__dirname, '..', 'gyp', 'pylib')
+    process.env.PYTHONPATH = path.resolve(__dirname, '..', 'gyp', 'pylib') + ':' + process.env.PYTHONPATH
 
     var cp = gyp.spawn(python, argv)
     cp.on('exit', onCpExit)
