From f38147792fde04e1b223216ca6283777eb2022ff Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tom=C3=A1=C5=A1=20Hr=C4=8Dka?= <thrcka@redhat.com>
Date: Fri, 30 Jan 2015 08:56:50 +0100
Subject: [PATCH] Use system gyp

---
 lib/configure.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/configure.js b/lib/configure.js
index 68f26d6..6f4fd40 100644
--- a/lib/configure.js
+++ b/lib/configure.js
@@ -297,7 +297,7 @@ function configure (gyp, argv, callback) {
     })
 
     // this logic ported from the old `gyp_addon` python file
-    var gyp_script = path.resolve(__dirname, '..', 'gyp', 'gyp_main.py')
+    var gyp_script = '/opt/rh/v8314/root/usr/bin/gyp'
     var addon_gypi = path.resolve(__dirname, '..', 'addon.gypi')
     var common_gypi = path.resolve(nodeDir, 'common.gypi')
     var output_dir = 'build'
-- 
1.8.3.1

