From 98b7b50acf8aba204c25ca10672b0e318467e192 Mon Sep 17 00:00:00 2001
From: Tomas Hrcka <thrcka@redhat.com>
Date: Mon, 30 Nov 2015 10:17:06 +0100
Subject: [PATCH] ides

---
 lib/configure.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/configure.js b/lib/configure.js
index 4e06529..a446943 100644
--- a/lib/configure.js
+++ b/lib/configure.js
@@ -247,7 +247,7 @@ function configure (gyp, argv, callback) {
     }
 
     // this logic ported from the old `gyp_addon` python file
-    var gyp_script = path.resolve(__dirname, '..', 'gyp', 'gyp_main.py')
+    var gyp_script = '/opt/rh/v8314/root/usr/bin/gyp'
     var addon_gypi = path.resolve(__dirname, '..', 'addon.gypi')
     var common_gypi = path.resolve(nodeDir, 'include/node/common.gypi')
     fs.stat(common_gypi, function (err, stat) {
-- 
2.5.0
>>>>>>> rhscl-2.1-nodejs010-rhel-6

